package.json

```diff
@@ -1,15 +1,15 @@
 {
   "name": "@nypl/dgx-header-component",
-  "version": "2.4.19",
+  "version": "2.5.0",
   "description": "NYPL Header as a React component.",
   "main": "dist/components/Header/Header.js",
   "scripts": {
     "start": "NODE_ENV=development node server.js",
     "babel-build": "./node_modules/.bin/babel src -d dist && cp -r src/styles/ dist/styles",
     "dist": "NODE_ENV=production ./node_modules/.bin/webpack --config webpack.config.js && cp -r src/styles/ dist/styles",
-    "test": "./node_modules/.bin/mocha test/helpers/browser.js test/*.test.js",
-    "test-file": "node_modules/.bin/mocha -w test/helpers/browser.js",
-    "test-with-coverage": "./node_modules/.bin/nyc ./node_modules/.bin/mocha test/helpers/browser.js test/*.test.js",
+    "test": "IS_TEST_ENV=true ./node_modules/.bin/mocha test/helpers/browser.js test/*.test.js",
```

Comment:
I made `IS_TEST_ENV` as an env variable when we run test. The coupled scripts to read it are in line 17 and 137 in test/EncoreLogOutTimer.test.js